config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json

```diff
@@ -1,94 +1,95 @@
 {
+  "id": "01FXGFB7J8MQJWRZ19R223H1TC",
   "name": "upper-hutt_urban_2017_0-10m",
-  "id": "im_01F6P1XK2AVYDJ66NY2MKSEXW1",
-  "uri": "s3://linz-basemaps/2193/upper-hutt_urban_2017_0-10m/01F6P1XK2AVYDJ66NY2MKSEXW1",
-  "year": 2017,
   "projection": 2193,
-  "resolution": 100,
+  "uri": "s3://linz-basemaps/2193/upper-hutt_urban_2017_0-10m/01FXGFB7J8MQJWRZ19R223H1TC",
+  "year": 2017,
+  "resolution": 90,
```

Comment:
@Wentao-Kuang this resolution looks odd?

Reply:
The resolution get from `CogStacJob.output.quality`. Weird, looks different from the setting.

Reply:
I might doing the wrong way, Should use `extractResolutionFromName`. I will update that in code.

Reply:
`quality` is the JPEG/Webp quality right? so `extractResolutionFromName` would be better?